### ISSUE TITLE
[accessibility issue] Auto refresh dropdown in Dashboard header is not keyboard accessible

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#79633]
**Steps to reproduce**:
Go to any Grafana instance dashboard. Navigate to the "Set auto refresh interval" button. Try to activate the dropdown with keyboard alone. It does not  matter if a refresh interval has been chosen prior or not.

**Actual Result**:
The dropdown doesn't appear. Note if you click on it with a mouse first, the dropdown box is able to navigated with keyboard as expected.

**Expected Result**
Spacebar or enter should activate the refresh dropdown.

**Comment by 51stDimension** [comment]
Hey, Would like to work on this issue.

**Comment by ckbedwell** [comment]
Hi @51stDimension, yes please do! Once you have a PR open, make sure you link to this issue so we can make sure it gets tracked and attributed to you correctly 😄

**Comment by 51stDimension** [comment]
Hey its my first time in open source. Need to connect with you once regarding this. Also any doc for my local setup?

**Comment by ckbedwell** [comment]
We have a readMe with a [contributing section you can find here](https://github.com/grafana/grafana/blob/main/README.md#contributing). This has documentation of how to contribute and how to set-up your local dev set-up, as well as a link to our community Slack which is a great place to get help too. 😃

**Comment by tskarhed** [comment]
@ckbedwell I think this is fixed now? I can reproduce it, the RefreshPicker works well with the keyboard.

**Comment by ckbedwell** [comment]
I agree, looks fixed! 🥳
